Makefile

```diff
@@ -0,0 +1,36 @@
+SHELL := bash 
+TARGET ?= origin
+# pypy.org static page and blog makefile
+# type `make help` to see all options 
+
+all: build
+
+.PHONY: clean build help
+
+venv_nikola/bin/nikola:  ## create a virtualenv to build the website
+	@python3 -m pip install --upgrade pip
+	@python3 -m venv ./venv_nikola
+	@venv_nikola/bin/python -m pip install wheel
+	@venv_nikola/bin/python -m pip install -r requirements.txt
+	@venv_nikola/bin/nikola plugin -i localsearch
```

Comment:
I don't understand the purpose of having this here (or really of having the makefile at all). This command just creates plugins/localsearch. Why not just commit that to the repo?

Reply:
I just ran into the same thing. Sorry, I was in a hurry and didn't review this PR closely enough. Invoking `pip install` and downloading packages on every invocation of `make auto` is a big no-no. I'll open a separate issue for this.